client/capstone-app/src/pages/GetStarted.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from "react";
+import SearchResults from "../components/SearchResults";
+import { useNavigate, Link } from "react-router-dom";
+
+
+const GetStarted = () => {
+    const [currentLocation, setCurrentLocation] = useState("")
+    const [destination, setDestination] = useState("")
+    const [searchData, setSearchData] = useState(null)
+    const [currentData, setCurrentData] = useState(null)
+    const [destinationData, setDestinationData] = useState(null)
+    const [currentCoordinates, setCurrentCoordinates] = useState(null)
+    const [destinationCoordinates, setDestinationCoordinates] = useState(null)
+    const [error, setError] = useState(null)
+    const navigate = useNavigate()
+
+
+    const handleSubmit = () => {
+        console.log(currentLocation, ", ", destination)
+        console.log(currentCoordinates, ", ", destinationCoordinates)
+    }
+    const findCurrentAddress = async (location) => {
+        try {
+            const response = await fetch(`https://api.openrouteservice.org/geocode/search?api_key=5b3ce3597851110001cf6248d4d2e47b8dbe429bbfff21f28b793179&text=${location}&boundary.country=USA`, {
```

Comment:
Is the country == USA an assumption that you want to make in your call?

Reply:
Yes, based on the data I'm working with, addresses outside the US are not valid